src/Infrastructure/Bus/Event/SymfonySyncDomainEventPublisher.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace CodelyTv\Infrastructure\Bus\Event;
+
+use CodelyTv\Shared\Domain\Bus\Event\DomainEvent;
+use CodelyTv\Shared\Domain\Bus\Event\DomainEventPublisher;
+use function Lambdish\Phunctional\each;
+
+final class SymfonySyncDomainEventPublisher implements DomainEventPublisher
```

Comment:
Please, help my understand the design decisions made for this `SymfonySyncDomainEventPublisher` 🙂

* Have you discarded the option of composing this `SymfonySyncDomainEventPublisher` class with the `SymfonySyncEventBus` one and publishing to it while calling to the `publishRecorded` function? I don't really get why this `DomainEventPublisher` doesn't do anything but storing the events in local map without actually publishing them to the Symfony Event Bus, sorry :S
* Why is it called with the `Symfony` prefix? At the end of the day, it isn't nothing more than an `InMemory` implementation, doesn't it?